src/modules/reusable/components/Metadata/index.js

```diff
@@ -0,0 +1,251 @@
+import { Link } from "react-router-dom";
+import { Icon } from "@umich-lib/core";
+import { useSelector } from "react-redux";
+import { icons } from "../../../reusable/components/Icon";
+import {
+  SPACING,
+  MEDIA_QUERIES,
+  COLORS,
+  LINK_STYLES
+} from "../../umich-lib-core-temp";
+import { stringifySearchQueryForURL } from "../../../pride";
+import {
+  Expandable,
+  ExpandableChildren,
+  ExpandableButton
+} from "@umich-lib/core";
+
+const visuallyHiddenCSS = {
+  border: 0,
+  clip: "rect(0 0 0 0)",
+  height: "1px",
+  margin: "-1px",
+  overflow: "hidden",
+  padding: 0,
+  position: "absolute",
+  width: "1px"
+};
+
+export default function Metadata({ data, kind }) {
+  const isCondensed = kind === "condensed";
+  const metadataCSS = !isCondensed
+    ? {
+        [MEDIA_QUERIES.LARGESCREEN]: {
+          display: "grid",
+          gridTemplateColumns: "9rem 1fr",
+          gridColumnGap: SPACING["S"],
+          "dt:not(:first-of-type) + dd": {
+            paddingTop: SPACING["XS"]
+          }
+        }
+      }
+    : {
+        dt: {
+          ...visuallyHiddenCSS
+        },
+        "dt:not(:first-of-type) + dd": {
+          paddingTop: SPACING["XS"]
+        }
+      };
+
+  // Only show expandable if more than 5.
+  function expandable(desc) {
+    if (desc.length <= 5) {
+      return {
+        show: desc.length,
+        expandable: false
+      };
+    }
+
+    console.log("desc", desc);
+
+    return {
+      show: 4,
+      expandable: true
+    };
+  }
+
+  return (
+    <dl
+      css={{
+        ...metadataCSS,
+        "dt:not(:first-of-type)": {
+          paddingTop: SPACING["XS"]
+        }
+      }}
+    >
+      {data.map((d, i) => (
+        <Expandable key={"expandable-metadata-dt-dd-" + i}>
+          <dt
+            css={{
+              gridColumnStart: "1"
+            }}
+          >
+            {d.term}
+          </dt>
+          <ExpandableChildren show={expandable(d.description).show}>
+            {d.description.map(d => (
+              <dd
+                css={{
+                  gridColumnStart: "2",
+                  display: "flex",
+                  alignItems: "top"
+                }}
+              >
+                <Description data={d} />
+              </dd>
+            ))}
+          </ExpandableChildren>
+
+          {expandable(d.description).expandable && (
+            <dd
+              css={{
+                gridColumnStart: "2",
+                display: "flex",
+                alignItems: "top"
+              }}
+            >
+              <ExpandableButton
+                name={d.term}
+                count={d.description.length}
+                kind="secondary"
+                small
+                css={{
+                  marginTop: SPACING["XS"]
+                }}
+              />
+            </dd>
+          )}
+        </Expandable>
+      ))}
+    </dl>
+  );
+}
+
+function Description({ data }) {
+  if (Array.isArray(data)) {
```

Comment:
@bertrama For descriptions that have an order, such as the case with academic discipline, the back-end can send that as the following:

```
{
  term: "Academic Discipline",
  descriptions: [
    [
      {
        text: "Science",
        search: { ... }
      },
      {
        text: "Biology",
        search: { ... }
      },
      {
        text: "Zoology",
        search: { ... }
      }
    ]
  ]
}
```

Reply:
Sample record:
https://deploy-preview-221--umich-lib-search-dev.netlify.com/catalog/record/000675899
https://search.lib.umich.edu/catalog/record/000675899

Reply:
This has been implemented and working properly.